### DISCUSSION TITLE
Fallback language

### DISCUSSION THREAD

**Discussion by OfekA-IAI** [Alexandre-Fernandez/astro-i18n#38] [Ideas]
It's a common option to specify a fallback language so that if translations are not available in a certain language, the t() function attempts to get the values from the fallback language, for example en.json.

Is it something that can be added?

**Comment by Alexandre-Fernandez** [comment]
Great idea, I'll try to add that in the next major version.

**Reply by alex22379** [reply]
This package is amazing, but adding this would really top it off

**Comment by Alexandre-Fernandez** [comment]
Just finished adding this to the version 2.0 (unreleased for now).
